ownCloud.xcodeproj/xcshareddata/xcschemes/ownCloud File Provider.xcscheme

```diff
@@ -43,13 +43,43 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "59371D922241125700C6BC5B"
+               BuildableName = "iosapp.xctest"
+               BlueprintName = "iosapp"
+               ReferencedContainer = "container:ownCloud.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "59371DA02241126900C6BC5B"
+               BuildableName = "ownCloud Screenshots Tests.xctest"
+               BlueprintName = "ownCloud Screenshots Tests"
+               ReferencedContainer = "container:ownCloud.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "59056CA922414F3C00A18A22"
+               BuildableName = "ownCloudScreenshotsTests.xctest"
+               BlueprintName = "ownCloudScreenshotsTests"
+               ReferencedContainer = "container:ownCloud.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
       </Testables>
       <MacroExpansion>
          <BuildableReference
             BuildableIdentifier = "primary"
-            BlueprintIdentifier = "DCC6564520C9B7E300110A97"
-            BuildableName = "ownCloud File Provider.appex"
-            BlueprintName = "ownCloud File Provider"
+            BlueprintIdentifier = "233BDE9B204FEFE500C06732"
+            BuildableName = "ownCloud.app"
+            BlueprintName = "ownCloud"
```

Comment:
Why do you need to changer these values?

Reply:
I did not do it deliberately, those files are changed by Xcode or by Pods.